code/modules/client/preference/preferences.dm

```diff
@@ -2020,8 +2020,6 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 	return 1
 
 /datum/preferences/proc/copy_to(mob/living/carbon/human/character)
-	var/datum/species/S = all_species[species]
-	character.change_species(species) // Yell at me if this causes everything to melt
```

Comment:
This line was added by #3226, and getting rid of it may cause problems with Ragin' Mages. I would avoid changing any existing lines in this file.